api/xds_protocol.rst

```diff
@@ -311,22 +300,22 @@ An example EDS request might be:
     resource_names:
     - foo
     - bar
-    type_url: type.googleapis.com/envoy.api.v2.ClusterLoadAssignment
+    "@type": type.googleapis.com/envoy.config.endpoint.v3.ClusterLoadAssignment
```

Comment:
this is not quite correct i think

Reply:
looking through the docs im thinking maybe it should just be `type_url` but im also thinking that it should probs be the v3 version - ditto below

Reply:
It should be `type_url`, this isn't `Any`, it's a first class field.

Reply:
updated/reverted